### ISSUE TITLE
Raw XML in xish is escaped

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#10359]
|[<img alt="ralphm's avatar" src="https://avatars.githubusercontent.com/u/116741?s=50" width="50" height="50">](https://github.com/ralphm)| @ralphm reported|
|-|-|
|Trac ID|trac#449|
|Type|defect|
|Created|2003-12-27 02:37:23Z|

<details><summary>Searchable metadata</summary>

```
trac-id__449 449
type__defect defect
reporter__ralphm ralphm
priority__high high
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__ 
keywords__ 
time__1072492643000000 1072492643000000
changetime__1470047133496526 1470047133496526
version__ 
owner__dizzyd dizzyd
cc__dizzyd cc__ralphm
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="ralphm's avatar" src="https://avatars.githubusercontent.com/u/116741?s=50" width="50" height="50">](https://github.com/ralphm)|@ralphm commented|
|-|-|

```
#!html
<pre>
Class Element in xish.domish has a method addRawXml that inserts a instance of
SerializedXML into its list of children. Upon serialization of the Element
(tree), this SerializedXML instance is wrongly escaped.

SerializedXML derives from str, and in the serializers (both _Serializer and
_ListSerializer) the method serialize first checks for string-like objects and
later for SerializedXML. That last check is never reached because SerializedXML
/is/ a string-like object.

Solution: swap checks
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="dizzyd's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|dizzyd commented|
|-|-|

```
#!html
<pre>
Did as suggested in bug report; fixed.
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|GitHub commented|
|-|-|

In changeset 6d9cd4bc1ca1b1aed07de4b8e3166c44885931da
```
#!CommitTicketReference repository="" revision="6d9cd4bc1ca1b1aed07de4b8e3166c44885931da"
Merge pull request #449 from twisted/8610-rodrigc-filepath-py3

Author: hawkowl, rodrigc
Reviewer: Lukasa, glyph
Fixes: #8610

Fix twisted.python.filepath.setContent() on Windows with Python 3
```
